hotness/tests/test_consumer.py

```diff
@@ -0,0 +1,130 @@
+"""
+Unit tests for hotness.consumer
+"""
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+
+import mock
+
+import fedmsg.config
+
+from hotness import consumers
+
+
+class TestConsumer(unittest.TestCase):
+    def setUp(self):
+        self.bz = mock.patch('hotness.bz.Bugzilla')
+        self.bz.__enter__()
+        self.koji = mock.patch('hotness.buildsys.Koji')
+        self.koji.__enter__()
+
+        test_config = fedmsg.config.load_config()
+        test_config["hotness.cache"] = {
+            "backend": "dogpile.cache.null",
+        }
+        test_config['hotness.requests_retries'] = 0
+
+        class MockHub(mock.MagicMock):
+            config = test_config
+
+        self.consumer = consumers.BugzillaTicketFiler(MockHub())
+        import logging
+        logging.basicConfig(level=logging.DEBUG)
+
+    def tearDown(self):
+        self.bz.__exit__()
+        self.koji.__exit__()
+
+    def test_is_retired_negative(self):
```

Comment:
Any reason you opted to not use vcrpy to record real responses? I know you mentioned it at FLOCK.

Reply:
Yeah - I just had trouble getting it to work and then bailed for lack of time.